### PR TITLE
ENH: Added a list of mimetypes to download

### PR DESCRIPTION
**Pull request by mpioski**
The Firefox driver does'nt automatically download files, so I added a list of various mimetypes to support.

### CODE REVIEW

File: botcity/web/config.py [hunk 1]
```diff
@@ -1,2 +1,285 @@
 DEFAULT_SLEEP_AFTER_ACTION = 300
 DEFAULT_NAVIGATE_TIMEOUT = 60000
+FIREFOX_MIMETYPES_TO_DOWNLOAD = ['application/vnd.hzn-3d-crossword', 'video/3gpp', 'video/3gpp2',
```
Comment (hhslepicka) [inline botcity/web/config.py:3]: If this is specific to Firefox would you mind to move it to the `botcity/web/browsers/firefox.py` ?

**Review by hhslepicka — CHANGES_REQUESTED** [review]
Thanks for the contribution @matheuspioski . 
I added a question (possible change request if positive answer).
Do you happen to have the same for Chrome?

**Comment by hhslepicka** [comment]
Another issue is with the linter. We enforce PEP8 with flake8 and your PR is breaking some PEP8 rules. Could you also address them?
https://github.com/botcity-dev/botcity-framework-web-python/pull/2/checks?check_run_id=3612390631#step:5:7

**Comment by mpioski** [comment]
> Thanks for the contribution @matheuspioski .
> I added a question (possible change request if positive answer).
> Do you happen to have the same for Chrome?
Chrome already downloads without a mimetype.

File: botcity/web/config.py (hunk 1 above)
Comment (mpioski) [reply botcity/web/config.py]: Done!

**Review by hhslepicka — APPROVED** [review]

**Pull request merged**